src/main/scala/com/mozilla/telemetry/streaming/FederatedLearningSearchOptimizer.scala

```diff
@@ -64,10 +64,15 @@ object FederatedLearningSearchOptimizer extends StreamingJobBase {
         val docType = fields.getOrElse("docType", "").asInstanceOf[String]
         if ("frecency-update" == docType) {
           val ping = FrecencyUpdatePing(m)
-          if ((ping.payload.study_variation contains "training") && (ping.payload.bookmark_and_history_num_suggestions_displayed > -1)) {
+          if (((ping.payload.study_variation contains "training") ||
+               (ping.payload.study_variation.startsWith("dogfooding")))
+            && (ping.payload.bookmark_and_history_num_suggestions_displayed > -1)) {
+
+            val model_version  = if (ping.payload.model_version == -1) Int.MaxValue else ping.payload.model_version
```

Comment:
This still won't cause the datapoints to be included in the model update -- [the `reduceOption` clause in `addBatch`](https://github.com/mozilla/telemetry-streaming/blob/master/src/main/scala/com/mozilla/telemetry/learning/federated/FederatedLearningSearchOptimizerS3Sink.scala#L45) means it's taking the averages from a single minimum model number that passed the previous filter